src/client/components/MessageList/MessageList.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import MessageList from './MessageList.js';
+
+export default { title: 'MessageList', component: MessageList };
+
+const testData = [
+  {
+    username: 'Anna',
+    message: { name: 'Anna', message: 'Hey, how is it going?' },
```

Comment:
here you can see that this looks a little strange. You are adding the message itself in the message
```suggestion
    text: 'Hey, how is it going?' ,
```